tests/e2e/fixtures/cluster.py

```diff
@@ -46,7 +46,7 @@ def associate_iam_oidc_provider(cluster_name, region):
 
 
 def create_iam_service_account(
-    service_account_name, namespace, cluster_name, region, iam_policy_arns
+    service_account_name, namespace, cluster_name, region, iam_policy_arns=[], iam_role_arns=[]
```

Comment:
can u attach multiple roles to same service account ?

Reply:
no seems like its only one, will fix that

```
Usage: eksctl create iamserviceaccount [flags]

General flags:
      --attach-policy-arn strings           ARN of the policy where to create the iamserviceaccount
      --attach-role-arn string              ARN of the role to attach to the iamserviceaccount
```